tests/test_db.py

```diff
@@ -2039,30 +2053,30 @@ def test_add_multiple_enrollments(self):
         self.assertEqual(enrollment.end, datetime.datetime(2005, 1, 1, tzinfo=UTC))
         self.assertIsInstance(enrollment.individual, Individual)
         self.assertEqual(enrollment.individual.mk, mk)
-        self.assertIsInstance(enrollment.organization, Organization)
+        self.assertIsInstance(enrollment.organization, Group)
```

Comment:
Check against `Organization`?

Reply:
It returns a `Group` instead of an `Organization` because the foreign key is mapped to `Group`. Should I change it back to `Organization`?

Reply:
That's a good point. The enrollment is going to be linked to a group because we will have teams and organizations, so that might mean organization should be renamed to `group` in the future, isn't it?

Reply:
Yes, it will be a group so individuals can be enrolled in teams too.

Reply:
Ok, so, let's keep it this way for now but we will have to change the attribute `organization` to `group`, I think.